code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -320,6 +320,11 @@
 	origin_tech = "engineering=3;combat=1"
 	var/weaken = 0
 
+/obj/item/weapon/restraints/legcuffs/bola/throw_at(atom/target, range, speed, mob/thrower, spin)
+	if(!..())
+		return
+	playsound(src.loc,'sound/weapons/bolathrow.ogg', 75, 1)

```

Comment:
doesnt that play on hit though? the sound is like a whoosh of it flying through the air, not an impact sound